molr-commons/src/main/java/io/molr/commons/domain/ExecutionStrategy.java

```diff
@@ -6,7 +6,7 @@
 
 public enum ExecutionStrategy {
 
-    PAUSE_ON_ERROR, PROCEED_ON_ERROR, ABORT_ON_ERROR;
+    PAUSE_ON_ERROR, PROCEED_ON_ERROR, ABORT_ON_ERROR, ABORT_ON_ERROR_LENIENT;
     
     private static final Map<String, ExecutionStrategy> namesToStrategy = new HashMap<>();
```

Comment:
How about making this an immutable map?

```
private static final Map<String, ExecutionStrategy> namesToStrategy = Arrays.stream(ExecutionStrategy.values()).collect(toImmutableMap(..));
```

... or do a search in a loop in the ofName() method.